### DISCUSSION TITLE
create ifcClassificationReference

### DISCUSSION THREAD

**Discussion by epeter-** [IfcOpenShell/IfcOpenShell#1719] [Q&A]
I am failing to implement an IfcClassificationReference from python.
the code:
```
....
classification = ifc.createIfcClassification(guid(), Source = newClass["Source"], Name =newClass["Name"], Edition = newClass["Edition"], EditionDate = dt)

id = 1
for o in output:
    ide = ifc.createIfcIdentifier(str(id))
    id += 1
    classificationReference = ifc.createIfcClassificationReference( Name = o[0] , ReferencedSource = classification, Identification = str(id))
    association = ifc.createIfcRelassociatesClassification(guid(), OwnerHistory = oh, Name = o[0] + "_Classification", RelatedObjects = o[1], RelatingClassification = classificationReference)

ifc.write(new_file)
```

everything is working untill i try to add "Identification" as attribute to the classificationReference.

examining the ifc schema shows that the attribute "Identification" is available for the classificationReference

schema:
![image](https://user-images.githubusercontent.com/6928491/131891443-c7d4525d-2de2-4a4c-9636-320c18c532ec.png)

maybe someone can point me towards a solution?

**Comment by aothms** [comment]
It's probably just that your file is in a different schema. The attribute is named ItemReference is IFC2X3.

![afbeelding](https://user-images.githubusercontent.com/1096535/131904224-f4a4284a-b634-4606-950b-98b9f71b527e.png)

https://standards.buildingsmart.org/IFC/RELEASE/IFC4_1/FINAL/HTML/schema/ifcexternalreferenceresource/lexical/ifcclassificationreference.htm
